typescript/cli/src/context/strategies/chain/MultiChainResolver.ts

```diff
@@ -58,9 +59,9 @@ export class MultiChainResolver implements ChainResolver {
         return this.resolveRelayerChains(argv);
       case ChainSelectionMode.CORE_APPLY:
         return this.resolveCoreApplyChains(argv);
-      case ChainSelectionMode.ORIGIN_DESTINATION:
+      case ChainSelectionMode.DEFAULT:
       default:
-        return this.resolveOriginDestinationChains(argv);
+        return this.resolveRelayerChains(argv);
```

Comment:
do we still need the RELAYER enum if the default is that behaviour?

Reply:
makes sense going to remove it